apis/mattermost/v1alpha1/clusterinstallation_types.go

```diff
@@ -92,6 +92,9 @@ type ClusterInstallationSpec struct {
 	// Defines the probe to check if the application is ready to accept traffic.
 	// +optional
 	ReadinessProbe corev1.Probe `json:"readinessProbe,omitempty"`
+	// Specify deployment pull policy
```

Comment:
nit:
```suggestion
	// Specify deployment pull policy.
```
This will probably require to regenerate files as the description will change, feel free to ping me in DM.